src/derivative_utils.jl

```diff
@@ -273,9 +273,9 @@ mutable struct WOperator{IIP, T,
     end
 end
 function WOperator{IIP}(f, u, gamma; transform = false) where {IIP}
-    if isa(f, Union{SplitFunction, DynamicalODEFunction})
-        error("WOperator does not support $(typeof(f)) yet")
-    end
+    # if isa(f, Union{SplitFunction, DynamicalODEFunction})
+    #     error("WOperator does not support $(typeof(f)) yet")
+    # end
     mass_matrix = f.mass_matrix
```

Comment:
@ChrisRackauckas what was the reason WOperator cannot handle SplitFunctions? All split ode tests are passing locally on this branch. CI should reflect that after fixing autodiff error.

Reply:
ok. I'm uncommenting this. We'll take care of this issue when we get rid of WOperator entirely.